interface.go

```diff
@@ -204,7 +204,7 @@ const (
 // notifications from a Network.
 type Notifiee interface {
 	Listen(Network, ma.Multiaddr)      // called when network starts listening on an addr
-	ListenClose(Network, ma.Multiaddr) // called when network starts listening on an addr
+	ListenClose(Network, ma.Multiaddr) // called when network closes listening on an addr
```

Comment:
you probably want to say "stops listening", closes doesn't make much sense syntactically.